src/licensedcode/data/rules/biosl-4.0_1.yml

```diff
@@ -3,9 +3,9 @@ is_license_notice: yes
 relevance: 100
 notes: seen in https://raw.githubusercontent.com/davidtiefenthaler/BOPT-DemandForecast/main/README.md
 ignorable_copyrights:
-    - Copyright (c) 2009, 2018 Robert Bosch GmbH
+    - Copyright (c) 2009, 2018 Robert Bosch GmbH and its subsidiaries
```

Comment:
I have no idea why the crlf change made this test fail:
 tests/licensedcode/test_detection_validate.py::TestValidateLicenseExtended::test_validate_detect_biosl_4_0_1_rule

I can reproduce locally, and this change makes the test pass.

Reply:
@pombredanne I see from other PR failures that this is actually a breakage of the develop mainline. not sure why..

Reply:
This is unrelated to your code!
Sorry fr the confusion, this is fixed mnow.